.travis.yml

```diff
@@ -13,8 +13,8 @@ script: ["bundle exec rake validate", "bundle exec rake lint", "bundle exec rake
 matrix:
   fast_finish: true
   include:
-  - rvm: 1.9.3
-    env: PUPPET_GEM_VERSION="~> 3.0"
+  # - rvm: 1.9.3
+  #  env: PUPPET_GEM_VERSION="~> 3.0"
```

Comment:
For some reason travis-ci tests failed for this ruby 1.9.3 with the
error: json_pure requires Ruby version ~> 2.0. Not sure why.

On Tue, Aug 16, 2016 at 4:13 PM, Raphaël Pinson <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/camptocamp/puppet-archive/pull/70#discussion_r74942359>
> :
>
> > @@ -13,8 +13,8 @@ script: ["bundle exec rake validate", "bundle exec rake lint", "bundle exec rake
> >  matrix:
> >    fast_finish: true
> >    include:
> > -  - rvm: 1.9.3
> > -    env: PUPPET_GEM_VERSION="~> 3.0"
> > +  # - rvm: 1.9.3
> > +  #  env: PUPPET_GEM_VERSION="~> 3.0"
>
> Why are you commenting these lines?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/camptocamp/puppet-archive/pull/70/files/5922e98103db1c6d2652578cbb386a081c5eb875#r74942359>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABXrPLYykr9fpIdUX3bImBjparMth-qFks5qgcWLgaJpZM4JhRCz>
> .
>